src/Traits/AmountTrait.php

```diff
@@ -19,13 +19,17 @@ trait AmountTrait
     /**
      * The calculated fee amount.
      *
+     * @Groups({"get"})
+     *
      * @var string|null
      */
     protected $paymentFee;
 
     /**
      * The subtotal after fees.
      *
+     * @Groups({"get"})
```

Comment:
As far as I understand it, it is possible to provide either of total or subtotal, or both - indicating you either want your total to include or exclude the calculated fees, whereas if you intend on using both the request will fail if the fees calculated have changed.

(This trait might not be used for that purpose?)

Reply:
After much investigation - yes, trait is not used for those purposes.